packages/api/cms-api/src/user-permissions/dto/current-user.ts

```diff
@@ -0,0 +1,27 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+import { GraphQLJSONObject } from "graphql-type-json";
+
+import { CurrentUserInterface } from "../../auth/current-user/current-user";
+import { ContentScope } from "../interfaces/content-scope.interface";
+
+@ObjectType()
+export class CurrentUserPermission {
+    @Field()
+    permission: string;
+}
+
+@ObjectType()
+export class CurrentUser implements CurrentUserInterface {
+    @Field()
+    id: string;
+    @Field()
+    name: string;
+    @Field()
+    email: string;
+    @Field()
+    language: string;
+    @Field(() => [GraphQLJSONObject])
```

Comment:
It would be possible to use the real content scope DTOs here if we'd make the content scope DTO configurable in the module. However, I believe this would add a lot of complexity to the code. Is it worth it, or are we okay with using JSON here?

Reply:
I'm yet not sure if it's worth it. We should decide this in later stages of the implementation when we have more potential spots where it might be an improvement.

Reply:
For the other resolvers we made that configurable. Very complicated and potential for errors (eg the validation issue!). Maybe it was a mistake??

In any case, it should be consistent - either everywhere complicated and correct or everywhere untyped.

We can change that later.

Reply:
Since we use discrete objecte as Content Scopes (see https://github.com/vivid-planet/comet/pull/1223#discussion_r1301082833) it's even more complicated. I don't really see the point implementing something at this stage of development.